Source/WebKit/WebKitSwift/TextAnimation/TextAnimationManager.swift

```diff
@@ -41,7 +41,7 @@ import WebKitSwift
         delegate.containingViewForTextAnimationType().addSubview(self.effectView)
     }
     
-    @objc(addTextAnimationTypeForID:withStyleType:) public func beginEffect(for uuid: UUID, style: WKTextAnimationType) {
+    @objc(addTextAnimationTypeForAnimationID:withStyleType:) public func beginEffect(for uuid: UUID, style: WKTextAnimationType) {
```

Comment:
I think this should be `addTextAnimationForAnimationID` since you're adding the animation and not just what type it is